AK/Format.cpp

```diff
@@ -567,7 +570,13 @@ ErrorOr<void> FormatBuilder::put_hexdump(ReadonlyBytes bytes, size_t width, char
                 TRY(put_literal("\n"sv));
             }
         }
-        TRY(put_u64(bytes[i], 16, false, false, true, Align::Right, 2));
+        if (upper_case) {
```

Comment:
This branch could be done a single time instead of embedding it in the loop. Simply change the above declaration of `lowercase_lookup` and `uppercase_lookup` to be a single `lookup` that is initialized based on the bool. For example:

```c++
StringView lookup = upper_case ? "0123456789ABCDEF"sv : "0123456789abcdef"sv;
```

Is there a reason to not do this?